test/cmd/core.sh

```diff
@@ -1319,13 +1319,18 @@ run_namespace_tests() {
   kubectl create namespace my-namespace
   # Post-condition: namespace 'my-namespace' is created.
   kube::test::get_object_assert 'namespaces/my-namespace' "{{$id_field}}" 'my-namespace'
+
   # Clean up
   kubectl delete namespace my-namespace --wait=false
   # make sure that wait properly waits for finalization
   kubectl wait --for=delete ns/my-namespace
   output_message=$(! kubectl get ns/my-namespace 2>&1 "${kube_flags[@]}")
   kube::test::if_has_string "${output_message}" ' not found'
 
+  kubectl create namespace my-namespace
+  output_message=$(! kubectl delete namespace -n my-namespace --all 2>&1 "${kube_flags[@]}")
```

Comment:
suggest deleting all of a different cluster-scoped object instead, like PersistentVolumes, where the command can actually succeed

Reply:
Done